### PR TITLE
Adds md for proposal of weekly sessions

### PR DESCRIPTION
**Pull request by marknuzz**

### CODE REVIEW

File: KoansAndKatas/koanwiki.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+###Intro
+We're exploring more ways to help the team get a stronger foundation of F#.  
+
+###Goals
+There are some goals I would like to achieve, all of which are part of the greater vision of empowering others, which is a core value of RealtyShares:
```
Comment (danielAtRS) [inline KoansAndKatas/koanwiki.md]: Replace all references of "I" with "we", to emphasize this is a team effort, not a personal ambition.

File: KoansAndKatas/koanwiki.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+###Goals
+There are some goals I would like to achieve, all of which are part of the greater vision of empowering others, which is a core value of RealtyShares:  
+1. Language or library fluency: A session could serve to provide a better understanding of how to use a group of specific functions or operators. 
+2. Team knowledge of each other's strengths and weaknesses with functional programming. This helps with targeting further training and assignment of tasks.  
+3. Morale - This should be something that helps everyone remember that this is a great place to work, and feel valued on the team.
+
+###Session schedule and accessibility  
+Once every week, either Thursday or Friday, team members will meet in one of the conference rooms, or optionally online. The session must be fully user-friendly via Internet as we have remote people on the team. The session will exist on Google Calendar as a recurring weekly meeting. If this system proves popular and productive, we can even consider doing these more often than once per week.
+
```
Comment (danielAtRS) [inline KoansAndKatas/koanwiki.md:12]: Good.

File: KoansAndKatas/koanwiki.md (hunk 1 above)
Comment (marknuzz) [reply KoansAndKatas/koanwiki.md]: Oops, forgot to do that thanks

File: KoansAndKatas/koanwiki.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+**Machine-Learning**  
+https://github.com/c4fsharp/Dojo-Ham-Or-Spam
+https://github.com/c4fsharp/Dojo-Digits-Recognizer 
+https://github.com/c4fsharp/Dojo-Markov-Bot
+
+**Other**  
+https://github.com/c4fsharp/Dojo-Picasquez-and-Velasso (not a strong F# tutorial but allows for creativity)
+
+
```
Comment (danielAtRS) [inline KoansAndKatas/koanwiki.md:59]: Looks good to me. @HoraceGonzalez ?

First step would be to setup the calendar invites, and get some volunteers for our first problem or two!

**Pull request merged**